wiki/Get-Help.md

```diff
@@ -101,7 +101,7 @@ paste a screenshot
 ### Before you ask a general question
 
 * We expect that you have already **set up Oppia on your machine**, and it is successfully running. (If not, kindly do that first!)
-* Prepare a debugging doc following [this template](https://docs.google.com/document/d/1OBAio60bchrNCpIrPBY2ResjeR11ekcN0w5kNJ0DHw8). For more information on how to use debugging docs, see [this link](https://github.com/oppia/oppia/wiki/Debugging-Docs).
+* Prepare a debugging doc following [this template](https://docs.google.com/document/d/1OBAio60bchrNCpIrPBY2ResjeR11ekcN0w5kNJ0DHw8). For more information on how to prepare a debugging doc following [the guidelines provided on the wiki page](https://github.com/oppia/oppia-android/wiki/Frequent-Errors-and-Solutions#how-to-write-a-debugging-doc).
```

Comment:
...prepare a debugging doc, follow the guidelines provided on [this wiki page](...).

Reply:
Done updated.

Reply:
This appears to be resolved in the latest changes, so marking it as resolved to unblock merging.